### PR TITLE
flex-boxs demo

### PR DESCRIPTION
**Pull request by ThomasB98**

### CODE REVIEW

File: flex-demo/index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+</head>
+<body>
+    <div class="flex-container">
+        <div class="flex-item">1</div>
+        <div class="flex-item">2</div>
+        <div class="flex-item">3</div>
+    </div>
+
+    <div>
+        <label for="justify-content">Justify-content:</label>
```
Comment (findmycode33) [inline flex-demo/index.html:18]: wrap label and select in one div for all

File: flex-demo/script.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+
+
+document.getElementById("justify-content").addEventListener("change", function() {
+    const value = this.value;
```
Comment (findmycode33) [inline flex-demo/script.js:4]: directly assign this.value to below line

File: flex-demo/index.html (hunk 1 above)
Comment (ThomasB98) [reply flex-demo/index.html]: Done 😀

File: flex-demo/script.js (hunk 2 above)
Comment (ThomasB98) [reply flex-demo/script.js]: done 👍

**Pull request merged**